### DISCUSSION TITLE
Error running KONG on docker

### DISCUSSION THREAD

**Discussion by PatrikUnterstell** [Kong/kong#11417] [Help] [answered]
Hi everyone,
Im trying to run kong on docker with this codeline following the [kong site](https://docs.konghq.com/gateway/3.3.x/install/docker/#install-kong-gateway-with-a-database)

`docker run  -e "KONG_PROXY_ACCESS_LOG=/dev/stdout"  -e "KONG_ADMIN_ACCESS_LOG=/dev/stdout"  -e "KONG_PROXY_ERROR_LOG=/dev/stderr"  -e "KONG_ADMIN_ERROR_LOG=/dev/stderr"  -e "KONG_ADMIN_LISTEN=0.0.0.0:8001, 0.0.0.0:8444 ssl"  -e "KONG_PG_HOST=kong-database"  -e "KONG_DATABASE=postgres"   -e "KONG_PG_USER=kong"  -e "KONG_PG_DATABASE=kong"  -e "KONG_PG_PASSWORD=kong"  -e "KONG_NGINX_DAEMON=off"  -p 8000:8000 -p 8443:8443 -p 8001:8001 -p 8444:8444 kong:3.3.0`

But im getting this error

`Error: could not prepare Kong prefix at /usr/local/kong:`

My Postgres is running OK
 
Im running on 
`Ubuntu 20.04.2 LTS
Release: 20.04
Codename: focal`

with 

`Docker version 20.10.7, build f0df350`

tks for your time.

**Comment by samugi** [comment]
Does the returned error not include anything after `could not prepare Kong prefix at /usr/local/kong:`?
Can you try with a minimal `docker run` command that does not configure a database and does not set log paths like the following: `docker run   -e "KONG_ADMIN_LISTEN=0.0.0.0:8001, 0.0.0.0:8444 ssl"  -e "KONG_DATABASE=off"  -e "KONG_NGINX_DAEMON=off"  -p 8000:8000 -p 8443:8443 -p 8001:8001 -p 8444:8444 kong:3.3.0` then add the options back one by one to find which one might be causing the problem?

**Reply by PatrikUnterstell** [reply, marked as answer]
The problem was with docker version.
After i upgrade to last version (24.0.5) the error was solved.
Tks @samugi for your time!
